app/models/comment.rb

```diff
@@ -1,2 +1,12 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_post_comment_counter
+
+  private
+
+  def update_post_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
### optional

- [ ] You can also use `post.increment!(:comments_counter)` to increment the comments count and `post.decrement!(:comments_counter)`  to decrement your comment count